src/components/ImageCard/ImageCard.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { describe, expect, it } from 'vitest'
+import { screen } from '@testing-library/react'
+import { render } from '@/tests/customRender'
+import ImageCard from './ImageCard'
+
+describe('ImageCard', () => {
+    it('should render ImageCard component correctly with basic props', async () => {
+        render(
+            <ImageCard src="https://i.imgur.com/pEI5qWM.jpeg" alt="Cat 🐱" />
```

Comment:
Avoid making network requests in unit tests. Let's link to a local file

```suggestion
            <ImageCard src="Krizza.jpg" alt="Cat 🐱" />
```